src/NoFrixion.MoneyMoov/Models/PaymentRequests/PaymentRequestCreate.cs

```diff
@@ -262,7 +262,7 @@ public class PaymentRequestCreate : IValidatableObject, IPaymentRequest
     /// </summary>
     //[RegularExpression(@"^([a-zA-Z0-9 ]{6,18})$",
     //ErrorMessage = @"The recipient reference must be between 6 and 18 characters and can only contain alphanumeric characters and space.")]
-    [Obsolete("This field will be set automatically to ensure the best chance of matching a payin to a payment reuqest.")]
+    [Obsolete("This field has been deprecated and is no longer needed to reconcile payments.")]

```

Comment:
Would it be nice to mention that the recipient reference will be set automatically to reconcile payments.

Reply:
Yeah sure. Will update the text.